### PR TITLE
Allow to disable join / leave echos

### PR DESCRIPTION
**Pull request by TurBoss**
this allows to disable join leave messages from matrix begin shown in discord

### CODE REVIEW

File: src/matrixeventprocessor.ts [hunk 1]
```diff
@@ -302,7 +310,7 @@ export class MatrixEventProcessor {
                 const mxClient = this.bridge.getClientFactory().getClientAs();
                 avatarUrl = mxClient.mxcUrlToHttp(profile.avatar_url, DISCORD_AVATAR_WIDTH, DISCORD_AVATAR_HEIGHT);
             }
-        }
+        }disablePresence
```
Comment (Half-Shot) [inline src/matrixeventprocessor.ts]: Looks like a copy and paste mistake?

Comment (TurBoss) [reply src/matrixeventprocessor.ts]: oh sorry

**Comment by TurBoss** [comment]
oh another typo 

hell!

**Comment by Half-Shot** [comment]
Sweet. Thank you for fixing a long standing issue :D.

**Comment by TurBoss** [comment]
thanks!

**Pull request merged**